Desktop_Interface/isodriver.cpp

```diff
@@ -823,6 +828,9 @@ void isoDriver::frameActionGeneric(char CH1_mode, char CH2_mode)
     }
 
     QVector<double> CH1, CH2;
+    double CH1_avg = 0;
+    double samplesPerSymbol = 0;
+    int numSymbols = 0;
```

Comment:
The Android build warns that these three variables are unused.  Can you wrap these lines with the `DISABLE_SPECTRUM` preprocessor stuff?

Reply:
That fixed it.  Thanks!